src/main/resources/db/migrations/V001__CreateSchemaTables.sql

```diff
@@ -0,0 +1,171 @@
+-------------------------------------- Create types
+CREATE TYPE public.COURT_TYPE AS ENUM ('crown', 'magistrate', 'family');
+CREATE TYPE public.PARTICIPANT_TYPE AS ENUM ('witness','defendant');
+CREATE TYPE public.RECORDING_STATUS AS ENUM ('standby', 'initialisation', 'recording', 'finished', 'processing', 'available', 'future');
+CREATE TYPE public.RECORDING_ORIGIN AS ENUM ('pre', 'vodafone');
+CREATE TYPE public.AUDIT_LOG_SOURCE AS ENUM ('application', 'portal', 'admin', 'auto');
+CREATE TYPE public.AUDIT_LOG_TYPE AS ENUM ('create', 'update', 'delete', 'action');
+
+-------------------------------------- Create tables
+CREATE TABLE public.regions (
+    id UUID PRIMARY KEY NOT NULL,
+    name VARCHAR(100)
+);
+
+CREATE TABLE public.courts (
+    id UUID PRIMARY KEY NOT NULL,
+    court_type COURT_TYPE NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    location_code VARCHAR(25)
+);
+
+CREATE TABLE public.rooms (
+    id UUID PRIMARY KEY NOT NULL,
+    room VARCHAR(45) NOT NULL
+);
+
+CREATE TABLE public.court_regions(
```

Comment:
a better name for this pivot table would be `court_region` as a court is being mapped to a region.

Reply:
just to clarify for consistency, should we use singular form for tables that represent relationships between the entities and plural form for the entities? 
Or should we use the singular form for all table names?

Reply:
yep - singular form for tables that represent relationships between the entities and plural form for the entities